checkov/common/bridgecrew/integration_features/features/custom_policies_integration.py

```diff
@@ -72,5 +81,25 @@ def _get_policies_from_platform(self):
         policies = [p for p in policies if p['isCustom']]
         return policies
 
+    def post_runner(self, scan_reports):
+        if self.bc_cloned_checks.keys():
+            scan_reports.failed_checks = self.extend_records_with_cloned_policies(scan_reports.failed_checks)
+            scan_reports.passed_checks = self.extend_records_with_cloned_policies(scan_reports.passed_checks)
+            scan_reports.skipped_checks = self.extend_records_with_cloned_policies(scan_reports.skipped_checks)
+
+    def extend_records_with_cloned_policies(self, records):
+        new_records = deepcopy(records)
```

Comment:
I don't think you need this line and the `deepcopy` a few lines lower is already doing what you want to achieve. Always trying to avoid `deepcopy` as much as possible 😄

Reply:
i like idempotent methods, but the performance implication can be too harsh and we overwrite the attribute anyway

Reply:
@gruebel You're right about the implications of the deep copy, but in this case we'll run in an infinite loop if we run without copying it - 
We're adding a new record to the records list, and if we don't deep copy `records`, when adding a record to `new_records` it adds it to the  `records` list as well, which results in an infinite loop.